templates/mobile/base_mobile.html

```diff
@@ -0,0 +1,13 @@
+{% extends "templates/base.html" %}
+
+{% block head_extra %}
+<meta property="og:image" content="{{ ASSET_SERVER_URL }}782d0221-facebook-share.jpg"/>
+{% if level_2 == 'features' %}
+<style>
+@-webkit-keyframes "slideOut"{100%{left:-300px;}}@-moz-keyframes slideOut {100%{left:-300px;}}@keyframes "slideOut"{100%{left:-300px;}}@-webkit-keyframes "slideOutRight"{from{left:-300px;}to{right:0;}}@-moz-keyframes slideOutRight {from{left:-300px;}to{right:0;}}@keyframes "slideOutRight"{from{left:-300px;}to{right:0;}}@-webkit-keyframes "slideOutRight2"{from{left:0;}to{right:-300px;}}@-moz-keyframes slideOutRight2 {from{left:0;}to{right:-300px;}}@keyframes "slideOutRight2"{from{left:0;}to{right:-300px;}}@-webkit-keyframes "slideIn"{100%{left:0;}}@-moz-keyframes slideIn {100%{left:0;}}@keyframes "slideIn"{100%{left:0;}}@-webkit-keyframes "fadeIn"{from{opacity:0;}to{opacity:1;}}@-moz-keyframes fadeIn {from{opacity:0;}to{opacity:1;}}@keyframes "fadeIn"{from{opacity:0;}to{opacity:1;}}@-webkit-keyframes "fadeOut"{from{opacity:1;}to{opacity:0;}}@-moz-keyframes fadeOut {from{opacity:1;}to{opacity:0;}}@keyframes "fadeOut"{from{opacity:1;}to{opacity:0;}}@-webkit-keyframes "rotate"{from{-webkit-transform:rotate(0deg);}to{-webkit-transform:rotate(-180deg);}}@-moz-keyframes rotate {from{-moz-transform:rotate(0deg);}to{-moz-transform:rotate(-180deg);}}@keyframes "rotate"{from{transform:rotate(0deg);}to{transform:rotate(-180deg);}}@-webkit-keyframes "rotateRev"{from{-webkit-transform:rotate(-180deg);}to{-webkit-transform:rotate(0);}}@-moz-keyframes rotateRev {from{-moz-transform:rotate(-180deg);}to{-moz-transform:rotate(0);}}@keyframes "rotateRev"{from{transform:rotate(-180deg);}to{transform:rotate(0);}}@-webkit-keyframes "animatedBackground"{0%{-webkit-transform:scale(1, 1);-moz-transform:scale(1, 1);-ms-transform:scale(1, 1);-o-transform:scale(1, 1);transform:scale(1, 1);opacity:0;}100%{-webkit-transform:scale(1.1, 1.1);-moz-transform:scale(1.1, 1.1);-ms-transform:scale(1.1, 1.1);-o-transform:scale(1.1, 1.1);transform:scale(1.1, 1.1);opacity:1;}}@-moz-keyframes animatedBackground {0%{-moz-transform:scale(1, 1);transform:scale(1, 1);opacity:0;}100%{-moz-transform:scale(1.1, 1.1);transform:scale(1.1, 1.1);opacity:1;}}@keyframes "animatedBackground"{0%{-webkit-transform:scale(1, 1);-moz-transform:scale(1, 1);-ms-transform:scale(1, 1);-o-transform:scale(1, 1);transform:scale(1, 1);opacity:0;}100%{-webkit-transform:scale(1.1, 1.1);-moz-transform:scale(1.1, 1.1);-ms-transform:scale(1.1, 1.1);-o-transform:scale(1.1, 1.1);transform:scale(1.1, 1.1);opacity:1;}}</style>
```

Comment:
Yeah, we're reusing some of the animations. I'll make sure to audit it when the content is complete.